### ISSUE TITLE
_SQL_GetAllReports is not a cross platform script

### ISSUE THREAD

**Issue by hyndgrinder** [FeedTheCube/CogDoc#32]
The WITH statement in the GetAllReports query uses MSSQL specific language.  Need to alter to allow the query to run in all platforms.

**Comment by SinclairC** [comment]
Is this query otherwise valid for different content-stores? Do they all store reports in the same place, with the same column names?

**Comment by hyndgrinder** [comment]
Yes, as far as I know, the process to build the store is pretty standard across platforms.  I can show you the structure if you like.

**Comment by SinclairC** [comment]
Just a small question about the new SQL: Is CMOBJPROPS7 a typo, or is it always #7? Or are there six more before that which might also have the data in some cases? I guess seeing the whole structure would be helpful.

**Comment by hyndgrinder** [comment]
There are almost 100 props tables, but they are always in the same order, and they are not always relevant to the CMOBJECTS record.  For instance, the PROPS7 table is for reports, while the PROPS1 table is for Users (I beleive).  There are also PROPS tables for folders, shortcuts, modules, ... basically every object in the cognos content store has a PROPS table.

**Comment by SinclairC** [comment]
Sounds good. I was just making sure that we don't have to account for reports in different PROPS tables. Is there anything useful to us in any other the others that we might want to incorporate into our program later?

**Comment by hyndgrinder** [comment]
...perhaps.  I haven't finished going through them all yet.  The one property I wanted (Owner) doesn't seem to be populating.  Typical.
